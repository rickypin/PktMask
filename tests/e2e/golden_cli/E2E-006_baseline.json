{
  "test_id": "E2E-006",
  "name": "Anonymize + Mask",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "cli_args": [
    "--anon",
    "--mask"
  ],
  "output_hash": "842f73e19e66611188fa4827f93af6c8813e0978f1a9445a2dc80fac65f7958b",
  "timestamp": "2025-10-09T21:31:11.487051",
  "version": "1.0-cli",
  "note": "Generated using CLI only - completely decoupled from API"
}