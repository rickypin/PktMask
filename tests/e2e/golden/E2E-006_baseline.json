{
  "test_id": "E2E-006",
  "name": "Anonymize + Mask",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "config": {
    "dedup": false,
    "anon": true,
    "mask": true
  },
  "output_hash": "842f73e19e66611188fa4827f93af6c8813e0978f1a9445a2dc80fac65f7958b",
  "output_file_size": 2721,
  "packet_count": 14,
  "stats": {
    "packets_processed": 14,
    "packets_modified": 15,
    "duration_ms": 417.3099994659424,
    "stages": [
      {
        "name": "AnonymizationStage",
        "packets_processed": 14,
        "packets_modified": 14,
        "duration_ms": 5.567073822021484
      },
      {
        "name": "Mask Payloads",
        "packets_processed": 14,
        "packets_modified": 1,
        "duration_ms": 6.556987762451172
      }
    ]
  },
  "timestamp": "2025-10-09T19:39:46.545741",
  "version": "1.0"
}